### PR TITLE
[WIP] README.md update

### PR DESCRIPTION
**Pull request by jaytmiller**
Here's a start at `CALCLOUD's` README.md.  Updates or follow-on PR's are needed,  particularly for the memory model, blackboard, and storage gateway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,266 @@
...
+            archiving.
+
+- control: is organized by IPPPSSOOT and stores job error/retry information as
+           well as inputs for the job requirements neural network evaluation.
+
+- blackboard: stores information which is used on premises by the OWL GUI to
+           display job and processing information.
+
+A periodic lambda is used to force synchronization of the Storage
+Gateway between AWS and on permise NFS by flushing the cache.  Flush
```
Comment (bhayden53) [inline README.md:134]: typo "permise"

File: README.md [hunk 2]
```diff
@@ -1,25 +1,266 @@
...
+Other messages are used to request action:
+
+- cancel-ipppssoot : kill the job associated with that ipppssoot.
+
+- rescue-ipppssoot : re-submit this job while tracking retries.  if it
+                     previously failed due to memory, use a higher memory rung.
+
+- clean-ipppssoot : delete all inputs, control, outputs, and messages for ipppssoot
+
+- cancel-all, rescue-all, clean-all : shorthand for performing cancel, rescue,
```
Comment (bhayden53) [inline README.md:170]: there's also clean-ingested now, if we're being exhaustive

File: README.md [hunk 3]
```diff
@@ -1,25 +1,266 @@
...
+3. Python memory error (unreported)
+
+While they are reported to or by AWS Batch in this complex way, they are all
+handled identically.
+
+### Other Errors
+
+These are distinguished as either "retryable" or "not retryable".
+
+1. CannotInspectContainer  - this can result when AWS Batch does not inspect an
```
Comment (bhayden53) [inline README.md:239]: "and is a transient error related to EBS throughput and IOPS provisioning"

File: README.md (hunk 2 above)
Comment (jaytmiller) [reply README.md:170]: added

File: README.md [hunk 4]
```diff
@@ -1,25 +1,266 @@
...
+                             exited/stopped container before it is deleted automatically,
+
+2. DockerError             -
```
Comment (bhayden53) [inline README.md:242]: Not sure we have a good explanation for these, but here was the actual error message:

> DockerTimeoutError: Could not transition to created; timed out after waiting 4m0s

File: README.md (hunk 3 above)
Comment (jaytmiller) [reply README.md:239]: thanks!

File: README.md (hunk 4 above)
Comment (jaytmiller) [reply README.md:242]: Added

**Review by alphasentaurii — APPROVED** [review]

**Pull request merged**